src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/Doctrine/DoctrinePHPCRDriver.php

```diff
@@ -28,6 +33,107 @@ class DoctrinePHPCRDriver extends AbstractDoctrineDriver
     /**
      * {@inheritdoc}
      */
+    public function load(ContainerBuilder $container, MetadataInterface $metadata)
+    {
+        parent::load($container, $metadata);
+        $this->addResourceListeners($container, $metadata);
+    }
+
+    /**
+     * Add resource event listeners.
+     */
+    protected function addResourceListeners(ContainerBuilder $container, MetadataInterface $metadata)
+    {
+        $options = array_merge(
+            [
+                'default_parent_path' => null,
+                'autocreate' => false,
+                'force' => false,
+                'name_filter' => true,
+                'name_resolver' => true,
+            ],
+            $metadata->getParameter('options')
+        );
+
+        $createEventName = sprintf('%s.%s.pre_%s', $metadata->getApplicationName(), $metadata->getName(), 'create');
+        $updateEventName = sprintf('%s.%s.pre_%s', $metadata->getApplicationName(), $metadata->getName(), 'update');
+
+        if ($options['default_parent_path']) {
+            // default path listener
+            $defaultPath = new Definition(DefaultPathListener::class);
+            $defaultPath->setArguments([
+                new Reference($metadata->getServiceId('manager')),
+                $options['default_parent_path'],
+                $options['autocreate'],
+                $options['force']
+            ]);
+            $defaultPath->addTag('kernel.event_listener', [
+                'event' => $createEventName,
+                'method' => 'onPreCreate'
+            ]);
+
+            $container->setDefinition(
+                sprintf(
+                    '%s.resource.%s.doctrine.odm.phpcr.event_listener.default_path',
```

Comment:
Looks consistent with f.e. the ORMDriver: `printf('%s.repository.%s.class', $metadata->getApplicationName(), $metadata->getName())`